docker-storage-setup.sh

```diff
@@ -720,6 +719,21 @@ unit: sectors
 
 ${dev}1 : start=     2048, size=  ${size}, Id=8e
 EOF
+}
+
+create_partition_parted(){
+  local dev="$1"
+  parted $dev --script mklabel dos mkpart primary 0% 100%
```

Comment:
Where is it documented that parted accepts start and end value in %.  Can you point me to it